js/alerts.js

```diff
@@ -2,8 +2,55 @@ var root_url = window.location.protocol + '//' + window.location.host;
 var data,det_row;
 var row_exists = false;
 
+function filterOnClick() {
+    //to update the options according to revision selected to maintain unique index[testindex and platindex]
+    updateSelectors('rev',function() {
+        //after the options are updated according to revision, indexes can be picked
+        var rev = $('#rev').val();
+        var test = $('#test').val();
+        var platform = $('#platform').val();
+        var showall = 0;
+        var testIndex = $("select[id='test'] option:selected").index();
+        var platIndex = $("select[id='platform'] option:selected").index();
+        if ($('#checkbox').is(":checked")) {
+            console.log("checked");
+            showall = 1;
+        }
+        document.cookie = "platform = " + platform;
+        document.cookie = "test = " + test;
+
+        var href = "alerts.html";
+        var flag = '?';
+        if (rev && rev != '') {
+            href += flag + "rev=" + rev;
+            flag = '&';
+        }
+        if (showall && showall != '') {
+            href += flag + "showAll=" + showall;
+            flag = '&';
+        }
+        if (testIndex && testIndex != '') {
+            href += flag + "testIndex=" + testIndex;
+            flag = '&';
+        }
+        if (platIndex && platIndex != '') {
+            href += flag + "platIndex=" + platIndex;
+            flag = '&';
+        }
+        location.href = href;
+
+    });
+
+    
+}
 function loadSelectors() {
-    $.getJSON(root_url + "/getvalues", function (data) {
+
+    get_params = { 
+                'name': [ 'keyrevision' ],
+                'value': [ results['rev'] ]
+            }
```

Comment:
nit: the spacing here could be better, I would pull the values and curly brace back about 8 spaces.